### DISCUSSION TITLE
Is there any indicator in a model fit object that indicates that integration errors occurred?

### DISCUSSION THREAD

**Discussion by billdenney** [nlmixr2/nlmixr2#223] [Q&A]
For errors like the one below, is there any flag or indicator in the model for object that indicates an error like the one below occurred?  (Or any other integration errors?)

```
unhandled error message: EE:lsoda -- at t = 1.78099e-09 and step size _rxC(h) = 6.79394e-15, the
 corrector convergence failed repeatedly or
 with fabs(_rxC(h)) = hmin
 @(lsoda.c:890
```

**Comment by mattfidler** [comment]
In the `$messages` it talks about an increase in `atol`/`rtol` which is an indicator that integration errors have occurred (though not which ID they occurred on).
